src/util.rs

```diff
@@ -28,8 +39,24 @@ where
             }
         }
     }
+}
 
-    guarded_transmute_pod_vec_permissive(a)
+/// Adapt a sequence of bytes for reading contiguous values of type `T`,
+/// by swapping bytes if the given endianness is not native. If no
+/// swapping is needed, the same byte slice is returned.
+pub fn adapt_bytes<T, E>(bytes: &[u8], e: E) -> Cow<[u8]>
```

Comment:
"function is never used: `adapt_bytes`" in non-features build.

Reply:
Right, I noticed that as well. What's a bit upsetting is that even if I conditionally compile this function, I have to conditionally `use` a few other functions as well. It might be best to just `allow` this function to be unused. I'll see what I can do.